test/test.py

```diff
@@ -943,6 +954,15 @@ def test_single_quotes_in_single_quotes(self):
         # "a'b" == 'a'\''b'
         self.assertEqual(self.sh.run_command("prog quote '1\t"), "1'1\r\n")
 
+    def test_wordbreak_chars(self):
+        self.assertEqual(self.sh.run_command('prog break a\tc'), 'a:b:c\r\n')
+        self.assertEqual(self.sh.run_command('prog break a:b:\tc'), 'a:b:c\r\n')
+        self.assertEqual(self.sh.run_command('prog break a:b\tc'), 'a:b:c\r\n')
+        self.assertEqual(self.sh.run_command("prog break 'a\tc'"), 'a:b:c\r\n')
+        self.assertEqual(self.sh.run_command("prog break 'a:b\tc\t"), 'a:b:c\r\n')
+        self.assertEqual(self.sh.run_command('prog break "a\tc"'), 'a:b:c\r\n')
+        self.assertEqual(self.sh.run_command('prog break "a:b\tc\t'), 'a:b:c\r\n')
```

Comment:
What are the double-tab tests supposed to mean? Do I understand correctly that the `\t` means pexpect will send TAB to the subprocess and trigger completion? What exactly do the characters after `\t` mean then? Aren't they disregarded and never passed to the completer? And shouldn't two tabs trigger completion twice? Thanks.